src/loot_mover.py

```diff
@@ -0,0 +1,92 @@
+from src.cam import Cam
+from src.config.loader import IniConfigLoader
+from src.config.models import MoveItemsType
+from src.logger import Logger
+from src.ui.char_inventory import CharInventory
+from src.ui.chest import Chest
+from src.utils.custom_mouse import mouse
+
+
+def move_items_to_stash():
+    Logger().info("Move inventory items to stash")
+
+    inv = CharInventory()
+    chest = Chest()
+
+    if chest.is_open():
```

Comment:
I like to handle the bad case as soon as possible so we don't have so many open blocks. In your case, you can check if the chest is not open, then log and return.

Reply:
I made this change. Some of the younger devs on my team also handle things like this, I think the standards have just changed on me in the last 17 years and I need to get used to it.

Reply:
:)